src/drone_controller/input_layer/request_handler.py

```diff
@@ -0,0 +1,44 @@
+"""
+This module handles all requests. This will cover REST calls as well as native python calls.
+"""
+from src.drone_controller.calculation_layer.thrust_calculator import ThrustCalculatorQuadroCopter
+from src.drone_controller.exception.exceptions import UserInputError
+from src.drone_controller.input_layer.drone_state import DroneState
+from src.drone_controller.input_layer.drone_state_mapper import DroneStateMapper
+
+
+class RequestHandler:
+    """
+    Handles request from python code.
+    """
+
+    def __init__(self, aircraft_type):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205179)